cnf-certification-test/accesscontrol/suite.go

```diff
@@ -531,3 +536,67 @@ func TestSysPtraceCapability(env *provider.TestEnvironment) {
 		ginkgo.Fail(errMsg)
 	}
 }
+
+//nolint:dupl
+func TestNamespaceLimitRanges(env *provider.TestEnvironment) {
+	var namespacesMissingLimitRanges []string
+	ginkgo.By("Testing namespace limit ranges")
+
+	for _, put := range env.Pods {
+		// Look through all of the pods and compare their namespace to any potential
+		// limit ranges that exist for a certain namespace.
+		foundPodNamespaceLimit := false
+		for index := range env.LimitRanges {
+			// We are just checking for the existence of the limit range as of right now.
+			// Read more about the limit range object here:
+			// https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/
+			if put.Data.Namespace == env.LimitRanges[index].Namespace {
+				foundPodNamespaceLimit = true
+				break
+			}
+		}
+
+		if !foundPodNamespaceLimit {
+			namespacesMissingLimitRanges = append(namespacesMissingLimitRanges, put.String())
+			logrus.Debugf("Pod %s is running in a namespace that does not have a LimitRange applied.", put.String())
+		}
+	}
+
+	if n := len(namespacesMissingLimitRanges); n > 0 {
+		errMsg := fmt.Sprintf("Number of pods running in namespaces that do not have a LimitRange applied: %d", n)
+		tnf.ClaimFilePrintf(errMsg)
+		ginkgo.Fail(errMsg)
+	}
+}
+
+//nolint:dupl
+func TestNamespaceResourceQuota(env *provider.TestEnvironment) {
+	var namespacesMissingQuotas []string
+	ginkgo.By("Testing namespace resource quotas")
+
+	for _, put := range env.Pods {
```

Comment:
As _ResourceQuotas_ apply to namespace level, wouldn't it be more convenient to iterate through namespaces instead of pods?

Reply:
I think we only care about workload pods that are deployed in their namespaces.  It might be possible there are other namespaces in the cluster that are not holding pods that we are testing?

Reply:
That's right. I don't think the namespaces are labeled, so there is no way to distinguish the ones for running workloads from those for other purposes.